cpp/ql/lib/semmle/code/cpp/commons/Dependency.qll

```diff
@@ -34,7 +34,7 @@ class DependsSource extends Element {
   DependsSource() {
     // not inside a template instantiation
     not exists(Element other | this.isFromTemplateInstantiation(other)) or
-    // allow DeclarationEntrys of template specializations
+    // allow DeclarationEntries of template specializations
```

Comment:
`DeclarationEntry` is probably a term. I left this change in to be able to hang this comment. I expect to drop this change.

Reply:
I agree that
```
`DeclarationEntry`s
```
is preferable (rendered as `DeclarationEntry`s), the original is also OK.